airflow/settings.py

```diff
@@ -79,28 +79,26 @@
 json = json
 
 
-def policy(task_instance):
+def policy(task):
```

Comment:
Maybe add a type annotation here?

Reply:
Can add but this function needs to be overriden by users in their `airflow_local_settings.py` - so adding type annotation here won't have any impact I feel. WDYT?

Reply:
When the user copies the function, they'll also copy the annotation, makes the code more robust :)

Reply:
What version would you like/recommend:

1. With TYPE_CHECKING - https://github.com/apache/airflow/pull/7532/commits/fb4ede5523cb1e0aea8bff95766feffb54d5f0a5
2. Without TYPE_CHECKING - https://github.com/apache/airflow/pull/7532/commits/7776f8178d3b61cf003b10eccb7caa2be3f504a2

With (2) I am just a bit worried that there might be a chance for cyclic dependencies.

Reply:
I prefer 2, but if there is any chance of having cyclic dependencies we should just leave it out. The first one is a bit awkward.

Reply:
Ya it is causing issues: https://travis-ci.org/apache/airflow/jobs/654914183

Reply:
Removed Type Annotation

Reply:
Much better, sorry for the fuzz.

Reply:
No worries :)